dolphinscheduler-python/pydolphinscheduler/examples/yaml_define/Condition.yaml

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Define the process
+process:
+  name: "Condition"
+
+# Define the tasks under the process
+tasks:
+  - {"task_type": "Shell", "params": {"name": "pre_task_1", "command": "echo pre_task_1"}}
+  - {"task_type": "Shell", "params": {"name": "pre_task_2", "command": "echo pre_task_2"}}
+  - {"task_type": "Shell", "params": {"name": "pre_task_3", "command": "echo pre_task_3"}}
+  - {"task_type": "Shell", "params": {"name": "success_branch", "command": "echo success_branch"}}
+  - {"task_type": "Shell", "params": {"name": "fail_branch", "command": "echo fail_branch"}}
```

Comment:
Is there any way we can auto format the yaml? and add some check in our CI?

Reply:
@EricGao888 does spotless support yaml format?

Reply:
I can see it support in spotless readme
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/15820530/187334014-7a175b9e-6d86-4f2e-ac03-455a1bb48ce3.png">

Reply:
@zhongjiajie @jieguangzhou I think so. `Spotless` supports everything of official `prettier` and formatting `yaml` is a basic function of `prettier`. See https://github.com/diffplug/spotless/tree/main/plugin-maven#prettier for an example of using `Spotless` to format `Typescript` with `prettier`. I think formatting `yaml` is the same.

Reply:
No file changes when I run the command `./mvnw spotless:apply -pl dolphinscheduler-python`

<img width="681" alt="image" src="https://user-images.githubusercontent.com/31528124/187338036-3a85ee34-ebe5-415d-901f-39614ef2a6ec.png">

But if I add a space to the YAML File, the command above will fix the space. I am not sure if that means the YAML file has been formatted.

@zhongjiajie @EricGao888

Reply:
@jieguangzhou No, currently we haven't configured `Spotless` in DS to check and format `yaml`. To enable this feature, you need to add some configurations as I mentioned above in `pom.xml` for `Spotless`.

Reply:
@EricGao888 @zhongjiajie Thanks for your suggestion. I have used Spotless to format the YAML file.

I created an issue to discuss the YAML file format #11699